docs/cargo.md

```diff
@@ -65,4 +65,21 @@ podman build . \
   --tag my-rust-app
 ```
 
+## Limitations
+
+### Resolver v3 and MSRV-aware resolution
+
+Hermeto configures Cargo to work without requiring `rustc` in the container. To
+achieve this, Hermeto sets `CARGO_RESOLVER_INCOMPATIBLE_RUST_VERSIONS=allow`
+when running `cargo vendor`.
+
+**Impact**: None. Hermeto uses `cargo vendor --locked` which vendors the exact
+versions from your Cargo.lock file. Any MSRV-aware resolution choices you made
+when generating the lock file are fully preserved.
+
+**Note**: The only exception is PERMISSIVE mode when Cargo.lock is out-of-sync
+with Cargo.toml. In this case, Hermeto regenerates the lock file without
+MSRV-aware resolution, potentially selecting newer dependency versions than
```

Comment:
This is fair I guess.

Reply:
Yeah, it seemed like it wasn't possible to otherwise avoid invoking rustc unless we did this as a fallback